warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctions.java

```diff
@@ -22,6 +22,8 @@
     public static final String OPTIONS_FUNCTION = "options";
     public static final String UNIQUE_FUNCTION = "unique";
     public static final String GROUPBY_FUNCTION = "groupby";
+    public static final String INCLUDE_FUNCTION = "include";
+    public static final String EXCLUDE_FUNCTION = "exclude";
     
```

Comment:
Pretty sure this is the right place to put these constants, let me know if that needs to change.

Reply:
These are EvaluationPhaseFilterFunctions, not QueryFunctions

Reply:
and the functions are called includeRegex and excludeRegex